core/src/main/java/org/infinispan/health/HostInfo.java

```diff
@@ -0,0 +1,25 @@
+package org.infinispan.health;
+
+/**
+ * Information about the host.
+ *
+ * @author Sebastian Łaskawiec
+ * @since 9.0
+ */
+public interface HostInfo {
```

Comment:
Yes, indeed. Some information are already available via JMX. However the goal is to make this API available through REST (WildFly management REST interface). This way OpenShift will be able to monitor resource utilization and check the health status.